partner_identification/models/res_partner.py

```diff
@@ -125,15 +125,16 @@ def _inverse_identification(self, field_name, category_code):
             else:
                 raise ValidationError(
                     _(
-                        "This {record_name} has multiple IDs of this "
-                        "type ({category_code}), so a write via the "
-                        "{field_name} field is not possible. "
+                        "This %(record_name)s has multiple IDs of this "
+                        "type (%(category_code)s), so a write via the "
+                        "%(field_name)s field is not possible. "
                         "In order to fix this, please use the IDs tab."
-                    ).format(
```

Comment:
yes, OCA recommend it. we need to change to pass pre-commit

Reply:
Where it's recommended?

Reply:
since [odoo 14.0](https://github.com/odoo/odoo/pull/52155), `_()` has support for placeholders, so here the good move would be to change
```
                    % {
                        "record_name": record._name,
                        "category_code": category_code,
                        "field_name": field_name,
                    }
 ```

to

```            
record_name=record._name,
category_code=category_code,
field_name=field_name
```

Reply:
I'm sorry everyone because I don't read carefully. I have updated it

Reply:
> since [odoo 14.0](https://github.com/odoo/odoo/pull/52155), `_()` has support for placeholders, so here the good move would be to change
> 
> ```
>                     % {
>                         "record_name": record._name,
>                         "category_code": category_code,
>                         "field_name": field_name,
>                     }
> ```
> 
> to
> 
> ```
> record_name=record._name,
> category_code=category_code,
> field_name=field_name
> ```

Thanks for the link, I agree